dev_db/utils.py

```diff
@@ -5,29 +5,17 @@
 
 
 def get_max_id(m):
-    max_id = 0
-    id_primary_key = model_has_id_primary_key(m)
-    if id_primary_key:
-        instances = list(m._default_manager.all().order_by("-id")[:1])
-        if instances:
-            max_id = getattr(instances[0], "id", 0)
-    return max_id
+    last = m._default_manager.order_by("-pk").first()
+    return last.pk if last and isinstance(last, int) else 0
 
 
 def get_field_names(m):
     field_names = [f.name for f in m._meta.fields]
     return field_names
```

Comment:
We could simplify a bit. I think it's just as clear:

```suggestion
    return [f.name for f in m._meta.fields]
```